src/cli/flash.js

```diff
@@ -37,6 +37,14 @@ module.exports = ({ commandProcessor, root }) => {
 			},
 			'port': {
 				describe: 'Use this serial port instead of auto-detecting. Useful if there are more than 1 connected device. Only available for serial'
+			},
+			'tachyon' : {
+				boolean: true,
+				description: 'Flash Tachyon'
+			},
+			'verbose' : {
```

Comment:
You don't need to add `verbose` here. It's a global flag available to all commands already https://github.com/particle-iot/particle-cli/blob/master/src/app/cli.js#L42